ci/github/helpers/install_rclone.bash

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+#  Installs the latest version of rclone plugin
+#
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit   # abort on nonzero exitstatus
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don't hide errors within pipes
+IFS=$'\n\t'
+
+
+# Installation instructions from https://rclone.org/docker/
+
+apt-get -y install fuse
```

Comment:
there is no version pinned down here. Are you sure about that? which version is installed in the deployment machines?

Reply:
I did not pin a version, we may do that. The latest version from ubuntu apt-get (the one thus installed on master) is (2.9.9-3). We may pin it at that, if so, notify me. Then I will do so in the ansible playbooks as well.